applications/nrf5340_audio/src/bluetooth/le_audio_bis_headset.c

```diff
@@ -26,8 +26,9 @@ BUILD_ASSERT(CONFIG_BT_AUDIO_BROADCAST_SNK_STREAM_COUNT <= 2,
 
 static struct bt_audio_broadcast_sink *broadcast_sink;
 
-static struct bt_audio_stream streams[CONFIG_LC3_DEC_CHAN_MAX];
-static struct bt_audio_stream *streams_p[ARRAY_SIZE(streams)];
+/* Headset supports only a single stream of audio */
+static struct bt_audio_stream audio_stream;
+static struct bt_audio_stream *audio_stream_p = &audio_stream;
```

Comment:
Nitpick: Can you change the names in bis_gateway as well to audio_streams and audio_streams_p, so that we use the same names in every le_audio_(...) file?